lib/ex_webrtc/dtls_transport.ex

```
@@ -93,31 +97,67 @@ defmodule ExWebRTC.DTLSTransport do
     dtls
   end
 
-  def process_data(dtls, data) do
+  def process_data(dtls, <<f, _::binary>> = data) when f in 20..63 and not dtls.finished do
     case ExDTLS.process(dtls.client, data) do
       {:handshake_packets, packets} when dtls.ice_state in [:connected, :completed] ->
         :ok = ICEAgent.send_data(dtls.ice_agent, packets)
-        dtls
+        {:ok, dtls}
 
       {:handshake_packets, packets} ->
         Logger.debug("""
         Generated local DTLS packets but ICE is not in the connected or completed state yet.
         We will send those packets once ICE is ready.
         """)
 
-        %__MODULE__{dtls | buffered_packets: packets}
+        {:ok, %__MODULE__{dtls | buffered_packets: packets}}
 
-      {:handshake_finished, _keying_material, packets} ->
+      {:handshake_finished, keying_material, packets} ->
         Logger.debug("DTLS handshake finished")
         ICEAgent.send_data(dtls.ice_agent, packets)
-        %__MODULE__{dtls | finished: true}
+        dtls = setup_srtp(dtls, keying_material)
+        {:ok, %__MODULE__{dtls | finished: true}}
 
-      {:handshake_finished, _keying_material} ->
+      {:handshake_finished, keying_material} ->
         Logger.debug("DTLS handshake finished")
-        %__MODULE__{dtls | finished: true}
+        dtls = setup_srtp(dtls, keying_material)
+        {:ok, %__MODULE__{dtls | finished: true}}
 
       :handshake_want_read ->
-        dtls
+        {:ok, dtls}
     end
   end
+
+  def process_data(dtls, <<f, _::binary>> = data) when f in 128..191 and dtls.finished do
+    case ExLibSRTP.unprotect(dtls.srtp, data) do
+      {:ok, payload} ->
+        {:ok, dtls, payload}
+
+      {:error, reason} = err when reason in [:reply_old, :reply_fail] ->
```

Let's ignore all errors for now. Maybe just log warning so we see them